src/main/java/no/skatteetaten/aurora/mvc/AuroraHeaderRestTemplateCustomizer.java

```diff
@@ -32,22 +31,25 @@ public void customize(RestTemplate restTemplate) {
     }
 
     private void addCorrelationId(HttpRequest request) {
-        String id = RequestKorrelasjon.getId();
-        if (id == null || id.isEmpty()) {
-            id = UUID.randomUUID().toString();
-            RequestKorrelasjon.setId(id);
+        request.getHeaders().addIfAbsent(KORRELASJONSID_FIELD, getKorrelasjonsid());
+    }
+
+    private String getKorrelasjonsid() {
```

Comment:
Stusser på at man oppretter Korrelasjonsid to steder i koden (skjer også i AuroraRequestParser). Duplisering bør vel unngås.